### DISCUSSION TITLE
reverse slider not working

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#18032] [Q&A] [answered]
https://user-images.githubusercontent.com/78441708/166115757-5614bd5d-a285-4bcf-9238-08d221370eb2.mp4

it changes the shape drastically 

map:
[Duskfox & Vixage - Aurora (Axilotl).zip](https://github.com/ppy/osu/files/8597492/Duskfox.Vixage.-.Aurora.Axilotl.zip)
@ 01:32:600

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/18042
